webvtt.go

```diff
@@ -190,6 +222,18 @@ func ReadFromWebVTT(i io.Reader) (o *Subtitles, err error) {
 
 			// Append item
 			o.Items = append(o.Items, item)
+
+		case strings.HasPrefix(line, webvttTimestampMap):
+			if len(item.Lines) > 0 {
+				err = fmt.Errorf("Found timestamp map after processing subtitle items")
```

Comment:
Can you use `errors.New` instead, prefix error with `astisub: ` and start with a lower case letter ? `astisub: found timestamp map after processing subtitle items`